src/components/EmailTemplateClient.tsx

```
@@ -0,0 +1,49 @@
+
+  return (
+    <div
+      id="email-template-client"
+      style={{
+        ...defaultStyle,
+        ...data.style,
+      }}
+    >
+      {body && Array.isArray(body) && body.length > 0 ? (
+        body.map((block: Block) =>
+          BlockRendererClient({ block, previewMode: 'preview', imageCollectionSlug }),
```

ESLint

[error] Expected "imageCollectionSlug" to come before "previewMode".

(perfectionist/sort-objects)